magento/inventory
# AreProductsSalableForRequestedQtyInterface Rework

Introduce new result and request interfaces. Rework AreProductsSalableForRequestedQtyInterface. Introduce new API Endpoint: /V1/inventory/are-product-salable-for-requested-qty/





### Description (*)


### Fixed Issues (if relevant)

1. none.

### Manual testing scenarios (*)

1. Create orders with all available product types and verify no errors occurred.
2. Verify new API endpoint inventorySalesApiAreProductsSalableForRequestedQtyV1


### Questions or comments


### Contribution checklist (*)
